Frends.MicrosoftSQL.BulkInsert/Frends.MicrosoftSQL.BulkInsert/BulkInsert.cs

```diff
@@ -121,18 +121,21 @@ private static async Task<long> ExecuteHandler(Options options, string tableName
                 sqlBulkCopy.BulkCopyTimeout = options.CommandTimeoutSeconds;
                 sqlBulkCopy.DestinationTableName = tableName;
                 sqlBulkCopy.SqlRowsCopied += (s, e) => rowsCopied = e.RowsCopied;
-                sqlBulkCopy.NotifyAfter = options.NotifyAfter;
 
-                await sqlBulkCopy.WriteToServerAsync(dataSet.Tables[0], cancellationToken).ConfigureAwait(false);
+                // Calculate the number of rows and set value for NotifyAfter
+                var rowCount = dataSet.Tables[0].Rows.Count;
+                sqlBulkCopy.NotifyAfter = rowCount > 0 ? Math.Max(1, rowCount / 10) : 1;
```

Comment:
Does it matter what integer the user is putting on the Options.NotifyAfter when hard coded values are used here?